src/Symfony/Component/AssetMapper/ImportMap/ImportMapManager.php

```diff
@@ -27,7 +27,7 @@ public function __construct(
         private readonly AssetMapperInterface $assetMapper,
         private readonly ImportMapConfigReader $importMapConfigReader,
         private readonly RemotePackageDownloader $packageDownloader,
-        private readonly PackageResolverInterface $resolver,
+        private readonly PackageResolverRegistry $resolverRegistry,
```

Comment:
Cannot do this without breaking BC i think

Reply:
I'd like to have a little explanation here then 😅

Reply:
that's a BC break indeed for people that use the class directly without the FWB integration